lib/writeSource.js

```diff
@@ -11,7 +11,7 @@ module.exports = writeSource;
 /**
  * @private
  */
-function writeSource(gltf, basePath, name, isEmbedded, callback) {
+function writeSource(gltf, basePath, name, isEmbedded, isImageEmbedded, callback) {

```

Comment:
Can you also change the names here to `embed` and `embedImage`.

Also throughout can you rename to `embedImages`?